src/machines/java/com/enderio/machines/client/rendering/blockentity/CapacitorBankBER.java

```diff
@@ -0,0 +1,316 @@
+package com.enderio.machines.client.rendering.blockentity;
+
+import com.enderio.EnderIO;
+import com.enderio.core.client.RenderUtil;
+import com.enderio.machines.common.blockentity.capacitorbank.CapacitorBankBlockEntity;
+import com.enderio.machines.common.blockentity.capacitorbank.DisplayMode;
+import com.enderio.machines.common.blockentity.multienergy.ICapacityTier;
+import com.mojang.blaze3d.vertex.*;
+import com.mojang.math.Axis;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.Font;
+import net.minecraft.client.renderer.LevelRenderer;
+import net.minecraft.client.renderer.LightTexture;
+import net.minecraft.client.renderer.MultiBufferSource;
+import net.minecraft.client.renderer.RenderType;
+import net.minecraft.client.renderer.blockentity.BlockEntityRenderer;
+import net.minecraft.client.renderer.blockentity.BlockEntityRendererProvider;
+import net.minecraft.client.renderer.texture.TextureAtlasSprite;
+import net.minecraft.core.BlockPos;
+import net.minecraft.core.Direction;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.inventory.InventoryMenu;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.Block;
+import org.joml.Matrix3f;
+import org.joml.Matrix4f;
+import org.joml.Vector2i;
+
+public class CapacitorBankBER implements BlockEntityRenderer<CapacitorBankBlockEntity> {
+
+    private static final ResourceLocation FULL_BAR = EnderIO.loc("block/capacitor_additionals/capacitor_bank_bar_full");
+    private static final ResourceLocation END_BAR = EnderIO.loc("block/capacitor_additionals/capacitor_bank_bar_end");
+    private static final ResourceLocation ENERGY_BAR = EnderIO.loc("block/capacitor_additionals/capacitor_bank_bar_energy");
+
+    private static final ResourceLocation IO_1x1 = EnderIO.loc("block/capacitor_additionals/1x1_full");
+    private static final ResourceLocation IO_FULL = EnderIO.loc("block/capacitor_additionals/full");
+    public CapacitorBankBER(BlockEntityRendererProvider.Context context) {
+
+    }
+
+    @Override
+    public void render(CapacitorBankBlockEntity blockEntity, float partialTick, PoseStack poseStack, MultiBufferSource bufferSource, int packedLight,
+        int packedOverlay) {
+
+        if (blockEntity.getLevel() != null) {
+            for (Direction direction: new Direction[] {Direction.NORTH, Direction.EAST, Direction.SOUTH, Direction.WEST}) {
```

Comment:
very minor, but is it needed to recreate this array every time?

Reply:
Could use Direction.valuees()

Reply:
no, that would include Up and down which would just be annoying later down the line to deal with. I'll create a static array having only those 4 in them